modular_framework_core/spawners/robot_controllers_spawner.py

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env python
 
-# Copyright 2019 Shadow Robot Company Ltd.
+# Copyright 2020 Shadow Robot Company Ltd.
```

Comment:
In these cases, you should have years separated by comma:
2019, 2020

Reply:
Fixed